src/plugins/arkeo/Arkeo.tsx

```diff
@@ -0,0 +1,145 @@
+import { ArrowForwardIcon } from '@chakra-ui/icons'
+import { Box, Button, Flex, Grid, Heading, Image, Link, useColorModeValue } from '@chakra-ui/react'
+import type { DefiType } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import { useTranslate } from 'react-polyglot'
+import ArkeoBg from 'assets/arkeo-bg.jpg'
+import NodeImage from 'assets/node.svg'
+import { Main } from 'components/Layout/Main'
+import { SEO } from 'components/Layout/Seo'
+import { RawText, Text } from 'components/Text'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { KeyManager } from 'context/WalletProvider/KeyManager'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { isMobile as isMobileApp } from 'lib/globals'
+import type { OpportunityId } from 'state/slices/opportunitiesSlice/types'
+
+import { FoxTokenHolders } from './FoxTokenHolders'
+import { LpCards } from './LpCards'
+import { StakingCards } from './StakingCards'
+
+type OpportunityReturn = {
+  [k in DefiType]: OpportunityId[]
+}
+
+const opportunities: OpportunityReturn = {
+  staking: [
+    'eip155:1/erc20:0xee77aa3fd23bbebaf94386dd44b548e9a785ea4b',
+    'eip155:1/erc20:0xc14eaa8284feff79edc118e06cadbf3813a7e555',
+    'cosmos:osmosis-1/slip44:118',
+    'cosmos:cosmoshub-4/slip44:118',
+  ],
+  lp: [
+    'eip155:1/erc20:0x470e8de2ebaef52014a47cb5e6af86884947f08c',
+    'cosmos:osmosis-1/ibc:gamm/pool/1',
```

Comment:
Create a new `cosmos:osmosis-1/ibc:gamm/pool/1`, fine in web for now since usage is isolated, we can move it in caip later on if we end up reusing it a lot